phing/build.yml

```diff
@@ -93,3 +89,5 @@ phpcs:
 project:
   local:
     uri: ${project.local.protocol}://${project.local.hostname}
+  ci:
+    uri: 127.0.0.1:8888
```

Comment:
Also, if you're open to it, perhaps these larger scale changes are best suited to a beta release, during which there's less promise of non-breaking changes and unforeseen issues can be ironed out - just a thought about how to move forward.